src/main/java/sana/Sana.java

```diff
@@ -58,29 +50,30 @@ public String doCommand(String userCommand) {
 
         try {
             switch (command) {
-                case "bye":
-                    return new ByeCommand().executeCommand(parsedCmd, userTasks);
-                case "list":
-                    return new ListCommand().executeCommand(parsedCmd, userTasks);
-                case "mark":
-                    return new MarkCommand().executeCommand(parsedCmd, userTasks);
-                case "unmark":
-                    return new UnmarkCommand().executeCommand(parsedCmd, userTasks);
-                case "todo":
-                case "event":
-                case "deadline":
-                    return new AddTask().executeCommand(parsedCmd, userTasks);
-                case "delete":
-                    return new DeleteCommand().executeCommand(parsedCmd, userTasks);
-                case "find":
-                    return new FindCommand().executeCommand(parsedCmd, userTasks);
-                default:
-                    throw new UnknownCommandException();
+            case "bye":
+                return new ByeCommand().executeCommand(parsedCmd, userTasks);
+            case "list":
+                return new ListCommand().executeCommand(parsedCmd, userTasks);
+            case "mark":
+                return new MarkCommand().executeCommand(parsedCmd, userTasks);
+            case "unmark":
+                return new UnmarkCommand().executeCommand(parsedCmd, userTasks);
+            case "todo":
+            case "event":
+            case "deadline":
+                return new AddTask().executeCommand(parsedCmd, userTasks);
+            case "delete":
+                return new DeleteCommand().executeCommand(parsedCmd, userTasks);
+            case "find":
+                return new FindCommand().executeCommand(parsedCmd, userTasks);
+            default:
+                throw new UnknownCommandException();

```

Comment:
——— For PR review ———
May I suggest the further use of OOP-styles -- package the case statements into the Command object itself. A `ByeCommand` object should contain the alias `"bye"` that trigger it's execution, for example.

Most commands would benefit from having multiple aliases that trigger their execution too, such as `bye, exit, quit` all being valid and recognized commands that executes the `ByeCommand` action. Writing all these aliases into a switch-case would prove troublesome, especially as the project scales and expands to include new commands and features, so packing them into their respective Command object would prove to be handy for future scaling.